content/posts/jellyfin-10-5-0.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Jellyfin Release - v10.5.0"
+subtitle: "Our biggest release yet"
+author: "Julien Machiels"
+githubusername: "MrTimscampi"
+date: 2020-03-07T08:11:42+01:00
+justify: "center"
+---
+
+A myriad of improvements, bugfixes and a look towards the future.
+
+<!--more-->
+
+With **over 200 contributions** and **over 500 issues closed**, this is our biggest release yet. While it is a few months behind the scheduled Christmas deadline, our anniversary release packs a lot of new features.
+
+<a href="/downloads/" class="button button__accent">Download Jellyfin 10.5.0</a> <a href="https://github.com/jellyfin/jellyfin/releases/tag/v10.5.0" class="button hero__button">Read the full release notes</a>
+
+This post will feature some of the main enhancements available in Jellyfin version 10.5.0. For an exhaustive list of all the changes, you can find a complete list on [GitHub](https://github.com/jellyfin/jellyfin/releases/tag/v10.5.0).
+
+## Web client
+
+This cycle, part of our focus was on improving the look and feel of the web client.
+
+<div class="juxtapose">
+    <img data-label="10.4.3" src="/images/posts/jellyfin-10-5-0/old-home.jpg" name="" />
+    <img data-label="10.5.0" src="/images/posts/jellyfin-10-5-0/new-home.jpg" name="" />
+</div>
+
+The web client now uses the **Noto Sans** font for all the languages we ship with. This makes for a more consistent experience and ensures that **multilingual libraries look unified**.
+
+Jellyfin 10.5.0 ships with the Latin, Greek, Chinese, Japanese, Korean, Arabic, Cyrillic, Hebrew, Vietnamese and Devanagari versions of the Noto font, optimized for the web.
+
+<div class="juxtapose">
+    <img data-label="10.4.3" src="/images/posts/jellyfin-10-5-0/old-details.png" name="" />
+    <img data-label="10.5.0" src="/images/posts/jellyfin-10-5-0/new-details.png" name="" />
+</div>
+
+The most noticeable visual change is the **new details page layout**. It puts the artwork of your library front and center while looking more modern and polished.
+
+We have also completely **overhauled SSA/ASS subtitle support**. Through the use of asm.js and Web Assembly, we now provide improved rendering for these formats, fixing some long-standing issues for anime lovers.
```

Comment:
**longstanding** is the correct word here, I made the same mistake.

Reply:
So, this is an interesting case: both are correct.

Basically, longstanding or long-standing depends on which style guide you adhere to. Both are valid and in current usage, but one may be rare than the other.

See here: https://www.cjr.org/language_corner/long-hyphen.php

Reply:
I've seen both as well, but @mark-monteiro mentioned that in my review and I saw that the word without the dash is in the dictionary, so I would go for it.

However, if you dug in it deeper than I did and you're sure that the dashed one is better, go ahead :)

Reply:
It's not better per se, but it's just as valid :)